orcid-core/src/main/java/org/orcid/core/adapter/impl/MapperFacadeFactory.java

```diff
@@ -261,23 +261,6 @@ public void registerSourceConverters(MapperFactory mapperFactory, ClassMapBuilde
     private class SourceMapper<T, U> extends CustomMapper<SourceAware, SourceAwareEntity<?>> {
 
         @Override
```

Comment:
Hey @wjrsimpson,

Please take a look at this, I'm thinking that we should map the source from DB to JAXB only, not the other way, and, in the managers, before storing the DB element we manually set the source.

Thanks